src/main/java/uk/gov/ons/census/casesvc/model/dto/CollectionCase.java

```diff
@@ -30,4 +30,5 @@
   private String fieldCoordinatorId;
   private String fieldOfficerId;
   private String ceExpectedCapacity;
+  private Boolean addressInvalid;
```

Comment:
should this be below private Boolean refusalReceived?

Reply:
they're different things. Are you strongly of the opinion that it should be moved?